.github/scripts/alt_text.sh

```diff
@@ -0,0 +1,10 @@
+grep -rn --include=\*.md "\!\[\]"  | # search for lines with "![]" empty alt tag, only in markdown files
```

Comment:
Would it be just a few more regex characters to also check for alt tags that have empty spaces? I think this might do it...
```suggestion
grep -rn --include=\*.md "\!\[\s*\]"  | # search for lines with "![]" empty alt tag, only in markdown files
```

Reply:
I bring this up because I suspect that I will be an offender of this error type and I wouldn't want to let myself accidentally evade the checker :)

Reply:
excellent call, I didn't even think about that as a fail case to include! Thanks for calling it out :)